### ISSUE TITLE
Literal HTML tags are scrubbed from Diaspora posts

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#8358]
Remote Diaspora post: https://diasp.org/posts/14711907#f6b959603ece0138bf2f047d7b62795e
Local Friendica post: https://friendica.mrpetovan.com/display/f6b959603ece0138bf2f047d7b62795e

Item source:
```xml
<comment>
  <author>urbanroman@diasp.org</author>
  <guid>f6b959603ece0138bf2f047d7b62795e</guid>
  <parent_guid>934c40203ec501388657124c77a41d08</parent_guid>
  <text>two ways to make superscripts in a post: 
1. with the &amp;lt;sup&amp;gt; and &amp;lt;/sup&amp;gt; tag; 
2. with unicode characters &amp;lt;Ctl-Shift-u2077&amp;gt; or &amp;lt;Ctl-Shift-u00B3&amp;gt; for examples.</text>
  <created_at>2020-03-02T16:15:31Z</created_at>
  <author_signature>M9u46Z03UFX8SWqG8icZup9CrDfqAFqihf4PKOi1mgiVq83oovKhEc4XErO5GHtQXd6wb8C4ielJr88QgMPtY0q1IzTPVqsbVeuexDCwb6egg5KD8YAy2uJpbS8+Z6EWYEBUks5eSGN9WFC+NRvn+6A4HkijJD+QYrVSAxvfLq4Orib1MKKZhCLucpKBQSuHg+meH56K2HqiTpWCschq4QfcrUAkhAnmjb9oOO4aQGv5N943miZsnFqTIaWguaDpT6yRmfFFP0DVqgo1/DOHOdCCVF2DRp0EOSANXqQ5bJ8Vv4/Vr2qrdkUOPjohntzWvuJxthc4g5qsL/GROrYhSsHcR7RYPzDAoCD2ozrhV1tuQJN9cx0F2C5QweesrM3nwRYSg8tzjhBYDQZimG8NJV8NcmrLSn+47wKZVdRT3ORhKpY5lgOVWNBmJq3/OF1zoy0Vu9zkEwDktaYb9vZQ5ocAeeBWdRBgxsmLfC/t4oxApj89Jo8Gm4oXL1fYGGdjI4qyKNlqVD26urKhb163YIdZoRW5G554IpHNmNgcZDN4i6/KaEr1UvrbS6HBy2hCModLue9GcTyGcXnZTKGnsEMPxMHp48fkRl0+batlPUY7tkGLq9062mpkBKnQpnwtlhhuDnZz6lD0YWPO8EjB7D9WLBKpLLFf+aWZYp7JUT4=</author_signature>
  <parent_author_signature>ixAUQKublGIMsd9E5ZV4Fm4g0bUnrlprHZrY29MQ5Q7ZruOc+hsTotxRqM0TnYM2GX3PPE6WVcuglNh+sxN4Nz9Qejh/urfKcP+Hus+Frc9K8Ie9JAgzGr6QPqnGSxB3J9LAHpxZzIwBz8Dw00dVZMcC/yTB178Fzq6kjDmbg8rTsk+fkpeNr6AXxvflxJV/w+RnLxqTzMlpBzP1lyErWSCj9SaEiphdqSMCd+BuzWRR7g9JUoYYn7IXhBmHS1cp9LTnXMwNyfUe09mI49UinSfwBtXNa0j79fLIoAPhfRNJC3K5n4Tx9f//0HwFT/LTI1nxZQHF0WhjexxWCvvfvi8oV087nYhfsYqx+Sj4WrTFTw/Y0I3ynqQuqF9Osr64arOegwJOTWE20IBo2w+O182Ub6f98LGUo9XumIVX0fc1jbmt2eQ75r7AdjGBRnerwGlxoRiJb3lNkx+OvkAHiO5UrNpxkUtnIEPiWiv0oPK/RHSg2RRadovvNsPXKbD2liLPRp1TBwMQQ815T8WRQKLfuPjQPXxO8f+5OwYzzeVnqdLpPj4c/d6eIlR+PUWZ3ZyvRQz1psnpgg8KiPANNhyW8/8E/xXyBOhkmgefr3E6Oy/XT32tOJw8eGt/SFr8PXMOhwSG3OQoipJ0J3NUorivHKxhx7LwAb5+CMYn6Qc=</parent_author_signature>
</comment>
```

**Comment by annando** [comment]
This shouts out for a new test case.

**Comment by MrPetovan** [comment]
Indeed, although I'm more used to @nupplaphil saying this.

**Comment by annando** [comment]
Possibly this is caused by us coding our first test units at work in the last days.

**Comment by nupplaphil** [comment]
That's the way, ah ha, I like it ;-) *singsing*

**Comment by MrPetovan** [comment]
Fixed by #8407
